quickwit-cli/Cargo.toml

```diff
@@ -14,6 +14,7 @@ name = "quickwit"
 path = "src/main.rs"
 
 [dependencies]
+async-trait = "0.1.51"
```

Comment:
nit: we tend to not pin a package up to the minor version: `async-trait = '0.1'`